lib/oauth.js

```diff
@@ -451,6 +453,7 @@ internals.Client.prototype._request = async function (method, uri, params, oauth
     // Calculate OAuth header
 
     const requestOptions = {
+        ...this._wreckOptions,
         headers: {
             Authorization: internals.Client.header(oauth)
         }
```

Comment:
Wouldn't you want also to be able to pass extra headers other than `Authorization`?

I was thinking of something like that to accomodate for it:

```js
const requestOptions = {
        ...this._wreckOptions,
        headers: {
            ...this._wreckOptions.headers,
            Authorization: internals.Client.header(oauth)
        }
};
```

Reply:
Not a use case I've seen. I don't mind it :-)